tests/test_other.py

```diff
@@ -2602,9 +2602,9 @@ def test_demangle_cpp(self):
   def test_demangle_malloc_infinite_loop_crash(self):
     self.run_process([EMXX, test_file('malloc_demangle_infinite_loop.cpp'), '-g', '-s', 'ABORTING_MALLOC', '-s', 'DEMANGLE_SUPPORT'])
     output = self.run_js('a.out.js', assert_returncode=NON_ZERO)
-    if output.count('Cannot enlarge memory arrays') > 4:
+    if output.count('Cannot enlarge memory arrays') > 5:
       print(output)
-    self.assertLess(output.count('Cannot enlarge memory arrays'),  5)
+    self.assertLess(output.count('Cannot enlarge memory arrays'),  6)
```

Comment:
what changed here?

Reply:
This is because the abort gets reported to stderr one more time by the node process because it is rethrown to the top level.  Previously we would call `process.exit` rather that rethrow to the outside node process.

This number is basically a record of amount times we repeat ourselves when an abort happens in JS :-/   We should probably address that.